src/index.html

```diff
@@ -9,10 +9,74 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/index.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./styles/variables.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./styles/stars.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./styles/card.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./styles/button.css"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <div class="imac-photo">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac-photo"
+          width="160"
+          height="134"
+        />
+      </div>
+      <div>
```

Comment:
Please remove everywhere these `div` wrappers just use classes in `p` tag that you use inside

Reply:
<img width="606" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/a5728cab-dc4b-4692-a5e9-639ce03ab083">
still not fixed